IdentityCore/xcconfig/identitycore__idlib__common.xcconfig

```diff
@@ -1,4 +1,4 @@
 PRODUCT_NAME = IdentityCore
 GCC_PREFIX_HEADER = src/IdentityCore.pch
 GCC_PRECOMPILE_PREFIX_HEADER = YES
-GENERATE_MASTER_OBJECT_FILE = YES
+GENERATE_MASTER_OBJECT_FILE = NO
```

Comment:
This will be breaking for partners taking dependency on us. We should reach out to them.

Reply:
Good catch. This one should not be committed. Fixed.